content/rs/references/rest-api/quick-start.md

```diff
@@ -0,0 +1,316 @@
+---
+Title: Redis Enterprise Software REST API quick start
+linkTitle: Quick Start
+description: Redis Enterprise Software REST API quick start
+weight: 20
+alwaysopen: false
+categories: ["RS"]
+aliases:
+---
+
+Redis Enterprise Software includes a REST API that allows you to automate certain tasks. This article contains examples of ways to send a Redis Enterprise Software REST API request.
+
+## Fundamentals
+
+No matter which method you use to send API requests, there are a few common concepts to remember.
+
+| Type | Description |
+|------|-------------|
+| [Authentication]({{<relref "/rs/references/rest-api#authentication">}}) | Use [Basic Auth](https://en.wikipedia.org/wiki/Basic_access_authentication) with your cluster username and password |
+| [Ports]({{<relref "/rs/references/rest-api#ports">}}) | All calls are made to port 9443 by default |
+| [Versions]({{<relref "/rs/references/rest-api#ports">}}) | Specify the version in the request [URI](https://en.wikipedia.org/wiki/Uniform_Resource_Identifier) |
+| [Headers]({{<relref "/rs/references/rest-api#headers">}}) | `Accept` and `Content-Type` should be `application/json` |
+| [Response types and error codes]({{<relref "/rs/references/rest-api#response-types-and-error-codes">}}) | A response of `200 OK` means success, otherwise, there was an error |  
```

Comment:
```suggestion
| [Response types and error codes]({{<relref "/rs/references/rest-api#response-types-and-error-codes">}}) | A response of `200 OK` means success; otherwise, the request failed due to an error |  
```